pandevice/userid.py

```diff
@@ -240,7 +253,7 @@ def register(self, ip, tags):
                 member.text = tag
         self.send(root)
 
-    def unregister(self, ip, tags):
+    def unregister(self, ip, tags, nonexistant=None):
```

Comment:
Updating the function signature to take missing arguments feels weird to me.  I think it would be better that the processing should be done before calling `unregister()`.

Also, just making sure, but if everything passed in is non-existent, does this code handle that properly?